R/mergeRaster.R

```diff
@@ -109,5 +109,9 @@ setMethod(
       }
       x[[i]] <- crop(r, extent(xminCut, xmaxCut, yminCut, ymaxCut))
     }
-    return(do.call(raster::merge, x))
+    y <- do.call(raster::merge, x)
+    yname <- gsub("_tile", "", names(x[[1]])) %>%
+      gsub("\\d", "", .)
```

Comment:
removes all numbers from the name, which isn't what we want. some rasters will have numbers in their names before splitting